cdisc_rules_engine/rules_engine.py

```diff
@@ -155,6 +166,7 @@ def validate_single_rule(
                 error_obj.domain = dataset_domain
                 return [error_obj.to_representation()]
         except Exception as e:
+            logger.trace(e, __name__)
```

Comment:
Do we need this trace?

Reply:
I would leave it here since when the error message is shown only when logging is set to "error".

Reply:
How is that different from having to set the logging level to "trace"

Reply:
The trace information is only shown when it is set to trace. I think it might be better to enable trace when the log level is set to error. Do we want to know the trace information when it has error?